bindings/generator/api/client.py

```diff
@@ -13,17 +13,17 @@
     Result,
     UserID,
     Variant,
-    VariantItemUnit,
 )
 from .config import ClientConfig
 from .events import OnClientEventCallback
+from enum import IntEnum
 
 
-class RealmRole(Variant):
-    Owner = VariantItemUnit
-    Manager = VariantItemUnit
-    Contributor = VariantItemUnit
-    Reader = VariantItemUnit
+class RealmRole(IntEnum):
+    Owner = 0
```

Comment:
Using the alternative form:

```python
OS = Enum(
    "OS",
    ["Linux", "MacOs", "Windows", "Android"],
)
```

result in:

```
Variable "bindings.generator.api.os.OS" is not valid as a type
```

when `Enum` is defined like so:

```python
class Enum(enum.Enum):
  pass
```

Otherwise, directly using `enum.Enum` don't produce that error.

Reply:
done